fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -192,7 +193,7 @@ public Triple apply(final Statement input) {
         }
         servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
-        return Response.ok(rdfStream).build();
+        return ok(rdfStream.withThisContext(rdfStream.transform(new Deskolemizer(idTranslator, null)))).build();
```

Comment:
I'm coming around to see this differently-- I now don't think this stuff is out of place in the HTTP layer. This is on the reasoning that bnodes are scoped entirely to the _document_ in which they appear, and there is no notion of document in our code below the HTTP layer… I think I could make this look better in the HTTP layer, but I really do think now that it belongs here.
I think that clients interacting at a deeper layer need to handle deskolemization themselves, _because only those clients themselves will know what the appropriate scope is in which to deskolemize_.
If this is to be handled in the kernel, what is the notion of scope in play? What is a "document" in the kernel? We operate only over streams of triples in the kernel…